Application Actions/Preview/1.0/default.json

```diff
@@ -0,0 +1,9 @@
+{
+  "RotateLeft"  : "Scroll Down",
+  "RotateRight" : "Scroll Up",
```

Comment:
If that's the case perhaps it would be better to just make a single commit to change the direction for all of the relevant applications instead of including it in this PR. What do you think?